### DISCUSSION TITLE
Fix warning "Files should be imported relative to their import location"

### DISCUSSION THREAD

**Discussion by realroot2185** [slint-ui/slint#5849] [Q&A] [answered]
I have this import `import { SideBar } from "ui/sidebar.slint"` in `ui/appwindow.slint`.

When I run `cargo run`:
`warning: app@0.1.0: Loading "ui/sidebar.slint" relative to the work directory is deprecated. Files should be imported relative to their import location`

I tried a few things how specify the path correctly?

**Comment by ogoffart** [comment, marked as answer]
since it is in `ui/appwindow.slint`,  the import should be relative to this file. So it should be:
`import { SideBar } from "sidebar.slint"` as sidebar.slint and appwindow.slint are in the same directory.

(Resolving through the current working directory used to work by mistake so that is why it is a warning.)
